examples/default/src/components/InputField.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { PropsWithChildren } from 'react';
+
+import { KeyboardTypeOptions, StyleSheet, TextInput } from 'react-native';
+
+interface InputFieldProps extends PropsWithChildren {
```

Comment:
I think there's no need to extend `PropsWithChildren` here since `InputField` won't have any children anyway.